test/TestAssets/TestAssets.sln

```diff
@@ -820,6 +822,18 @@ Global
 		{9B5BDF7D-5816-47C6-8CFD-E45B152CA35F}.Release|x64.Build.0 = Release|Any CPU
 		{9B5BDF7D-5816-47C6-8CFD-E45B152CA35F}.Release|x86.ActiveCfg = Release|Any CPU
 		{9B5BDF7D-5816-47C6-8CFD-E45B152CA35F}.Release|x86.Build.0 = Release|Any CPU
+		{64C4C4CD-2319-4033-9F33-53AAA8FECEFA}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{64C4C4CD-2319-4033-9F33-53AAA8FECEFA}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{64C4C4CD-2319-4033-9F33-53AAA8FECEFA}.Debug|x64.ActiveCfg = Debug|Any CPU
```

Comment:
Could you remove the extra added configurations that are not used/needed?

Reply:
I'll follow up for this one...this sln is redacted by a classic add project and is not cleaned yet.